platforms/documentation/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -608,6 +608,12 @@ Set to `rich` to enable color and other rich output in the console output, regar
 +
 Set to `verbose` to enable color and other rich output like `rich` with output task names and outcomes at the lifecycle log level, (as is done by default in Gradle 3.5 and earlier).
 
+[[sec:command_line_problems]]
+=== Reporting problems
+
+`--no-problems-report`::
+Disable the generation of `build/reports/problems-report.html`
```

Comment:
- Can we add a quick description of what is the default behavior and link the user guide? Something like "Gradle generates an html summary containing the problems reported via the problems API [link]"